sdk/python/packages/flet-core/src/flet_core/types.py

```diff
@@ -360,6 +360,38 @@ class StrokeJoin(Enum):
     BEVEL = "bevel"
 
 
+class ThemeVisualDensityDeprecated(EnumMeta):
+    def __getattribute__(self, item):
+        if item in [
+            "STANDARD",
+            "COMPACT",
+            "COMFORTABLE",
+            "ADAPTIVE_PLATFORM_DENSITY",
+        ]:
+            warn(
+                "ThemeVisualDensity enum is deprecated and will be removed in version 0.27.0. "
+                "Use VisualDensity enum instead.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+        return EnumMeta.__getattribute__(self, item)
+
+
+class ThemeVisualDensity(Enum, metaclass=ThemeVisualDensityDeprecated):
+    STANDARD = "standard"
+    COMPACT = "compact"
+    COMFORTABLE = "comfortable"
+    ADAPTIVEPLATFORMDENSITY = "adaptivePlatformDensity"
+    ADAPTIVE_PLATFORM_DENSITY = "adaptivePlatformDensity"
+
+
+class VisualDensity(Enum):
+    STANDARD = "standard"
+    COMPACT = "compact"
+    COMFORTABLE = "comfortable"
+    ADAPTIVE_PLATFORM_DENSITY = "adaptivePlatformDensity"
+
```

Comment:
why do we need two visual densities?

Reply:
I deprecated `ThemeVisualDensity` because it sounded theme-specific, and replaced it by simple `VisualDensity`.

Reply:
I see now, great!